spinalcordtoolbox/deepseg/models.py

```diff
@@ -19,23 +20,44 @@
 # TODO: add mirror
 MODELS = {
     't2star_sc':
-        {'url': 'https://osf.io/v9hs8/download?version=4',
-         'description': 'Cord segmentation on T2*-weighted contrast.',
+        {'url': 'https://osf.io/v9hs8/download?version=5',
+         'description': 'Cord segmentation model on T2*-weighted contrast.',
          'default': True},
     'mice_uqueensland_sc':
-        {'url': 'https://osf.io/nu3ma/download?version=5',
-         'description': 'Cord segmentation on mouse MRI. Data from University of Queensland.',
+        {'url': 'https://osf.io/nu3ma/download?version=6',
+         'description': 'Cord segmentation model on mouse MRI. Data from University of Queensland.',
          'default': False},
     'mice_uqueensland_gm':
-        {'url': 'https://osf.io/mfxwg/download?version=5',
-         'description': 'Gray matter segmentation on mouse MRI. Data from University of Queensland.',
+        {'url': 'https://osf.io/mfxwg/download?version=6',
+         'description': 'Gray matter segmentation model on mouse MRI. Data from University of Queensland.',
          'default': False},
     't2_tumor':
-        {'url': 'https://osf.io/uwe7k/download?version=1',
-         'description': 'Cord tumor segmentation on T2-weighted contrast.',
+        {'url': 'https://osf.io/uwe7k/download?version=2',
+         'description': 'Cord tumor segmentation model, trained on T2-weighted contrast.',
+         'default': False},
+    'findcord_tumor':
+        {'url': 'https://osf.io/qj6d5/download?version=1',
+         'description': 'Cord localisation model, trained on T2-weighted images with tumor.',
          'default': False}
     }
 
+# List of task. The convention for task names is: action_(animal)_(contrast)_region
+# Regions could be: sc, gm, lesion, tumor
+TASKS = {
+    'segment_t2star_sc':
```

Comment:
i find it a bit too verbose. I suggest to replace segment by seg.

also, I find it more intuitive to talk about the task, then object, then modality. So, something like: `seg_sc_t2star`. It could be the purpose of another PR, but since we are introducing a new TASKS section, it does not hurt to do it now.

Now, it also raises this question: if we implement the suggestion above, should we also propagate this logic to model names? this is more of a contentious point due to the large changes that it implies (i apologize in advance 😬). On the other hand, I don't find it shocking to have tasks called "seg_sc_t2" and models called "findcord_t2", "t2_sc", etc.

now, i am also realizing that we don't _really_ have a convention (yet) for our ivadomed models. For example:
- findcord_tumor → no modality is listed, cord instead of sc, and the task (find) is not separated with "_"
- t2star_sc → modality comes first, no task listed (seg). 

so, to minimize the amount of changes for this PR, maybe we can go with the above suggestion and only touch the TASK names, and for model names we open an issue on ivadomed and settle on a convention.

Reply:
Naming convention --> indeed, important points.
Change TASK names: I agree, happy to do this

Reply:
Note: when I named the TASK, I tried to match the[ convention that was stated for MODELS](https://github.com/neuropoly/spinalcordtoolbox/blob/master/spinalcordtoolbox/deepseg/models.py#L17), ie region last.

Reply:
Done in [7b86bb9](https://github.com/neuropoly/spinalcordtoolbox/pull/2748/commits/7b86bb96e228e1ac926cbdcba78cbe2d3151e2df)